tests/test_all.py

```diff
@@ -480,17 +480,24 @@ def test_forgotten_password(self):
 
     def test_resend(self):
         """Resend the invitation email to alpha invitee."""
+        # Date should get reset to today, so make sure original date is in the past.
+        self.alpha_invite.date_invited = self.old_date
+        self.alpha_invite.save()
         self.client.login(username='superuser', password='superuser')
         code = str(self.alpha_invite.activation_code)
         response = self.client.post(
             reverse('invite:resend', args=[code]),
             follow=True
         )
+        self.alpha_invite.refresh_from_db()
         self.assertEqual(200, response.status_code)
         self.assertIn(
             'Resent invitation email for {}'.format(self.alpha_invite.username),
             response.content.decode()
         )
+        self.alpha_invite.refresh_from_db()
```

Comment:
Do we need to `refresh_from_db` again here after doing it a few lines up?

Reply:
heh, nope, that was a mistake on my part, I'll fix it